.github/workflows/main-checks.yml

```diff
@@ -125,22 +125,38 @@ jobs:
           override: true
           profile: minimal
 
-      - uses: jetli/wasm-pack-action@v0.3.0
-        with:
-          version: "latest"
-
+      - uses: browser-actions/setup-geckodriver@latest
       - uses: Swatinem/rust-cache@v1
 
+      - name: Install wasm-bindgen-cli
+        shell: bash
+        run: |
+          if [ ! -f "Cargo.lock" ]; then
+            cargo fetch
+          fi
+          VERSION=`cargo run -p find-wasm-bindgen-version --quiet --release`
```

Comment:
would it be enough to run `cargo pkgid --frozen wasm-bindgen | cut -d ":" -f 3` here instead of writing your own tool?